src/qt/locale/bitcoin_sv.ts

```diff
@@ -2673,7 +2669,7 @@ Detta betyder att en avgift på minst %2 krävs.</translation>
     <message>
         <location line="+66"/>
         <source>Send Custom Message to a Gridcoin Recipient</source>
-        <translation type="unfinished"></translation>
+        <translation>Skicka ett meddelande till en Gridcoin mottagare</translation>
```

Comment:
Gridcoin-mottagare :)

Reply:
Apperently you found some särskrivningar.